lumicks/pylake/detail/imaging_mixins.py

```diff
@@ -47,7 +47,7 @@ def export_video(
             raise RuntimeError("You need either ffmpeg or pillow installed to export videos.")
 
         start_frame = start_frame if start_frame else 0
-        end_frame = end_frame if end_frame else self.num_frames - 1
+        end_frame = end_frame if end_frame else self.num_frames
```

Comment:
Probably, to be clear that `end_frame` is exclusive, name it `stop_frame` like the start and stop of indices?

Reply:
I fully agree. Analogously to before, I would vote for doing the name change only in the new `export_video()` if you agree.